test/intl402/RelativeTimeFormat/constructor/constructor/options-toobject-prototype.js

```diff
@@ -0,0 +1,35 @@
+// Copyright 2018 Igalia, S.L. All rights reserved.
```

Comment:
this `constructor/constructor/` path was not introduced in this PR but it's confusing. From the common idea of Test262 files structure, mostly from the built-ins folder, these are tests that could live directly on `test/intl402/RelativeTimeFormat/`.

In any case, after reviewing these tests, they look like they belong to the `test/intl402/RelativeTimeFormat/prototype/resolvedOptions` folder instead. We are checking results of calling it, even if the differences come from when the initial object is created.